### ISSUE TITLE
Desktop wallpaper setting issue

### ISSUE THREAD

**Issue by timrowledge** [gitbls/sdm#161]
This is simply a record of a problem noticed when changing settings; it almost certainly isn't an sdm problem but it is faintly possible that some interaction with the sdm process and Wayland is involved.

When using Wayland and wayfire to have a VNC connection, if the desktop wallpaper has been set to 'none (so that a plain background colour is used), then it appears that vnccannot connect properly. An error message is displayed - 
![Screen Shot 2024-01-12 at 3 50 42 PM](https://github.com/gitbls/sdm/assets/7818030/31fde57d-ff37-480c-ad6d-b93afa3be2a9)

Using ssh to log in and either delete the ~/.config/pcmanfm/desktop-items-0.conf files, or edit it to set 'wallpaper_mode=crop', followed by rebooting, seems to work. Changing the wallpaper mode setting from the running desktop works *until you reboot*.

Very strange, and TBH, somebody else's problem.

**Comment by gitbls** [comment]
Yes, saw that here as well. Was not able to fix it with wallpaper_mode=crop. VNC with wayland still has some newbie pains, apparently.

**Comment by timrowledge** [comment]
Just for completeness, tried on aPi 4, no different  

> On 2024-01-13, at 11:03 AM, Benn ***@***.***> wrote:
> 
> 
> Yes, saw that here as well. Was not able to fix it with wallpaper_mode=crop. VNC with wayland still has some newbie pains, apparently.
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.
> 


tim
--
tim Rowledge; ***@***.***; http://www.rowledge.org/tim
There *are* worse crimes than burning books.  One of them is not reading them.
